ci-scripts/copy_to_archive

```diff
@@ -40,18 +40,24 @@ fi
 if [ -d ${IMAGE_DIR}/${INTEL_ARCH} ]; then
     echo "Archiving Intel image to $ARCHIVE_DIR"
     mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
+    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
```

Comment:
Why is there a need for "*20*". 

only "-*.swu" would do the trick.

Reply:
but the "-" before 20 is enough to distinguish b/w symlink and the timestamped image. The symlink doesn't have "-".

Reply:
Please fix this one too

Reply:
In practice it is not working, I tried but it is still archiving both.